example/panel/modules/data_load_async_module.dart

```diff
@@ -12,32 +12,38 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-library w_module.example.panel.modules.dataLoadAsync_module;
+library w_module.example.panel.modules.data_load_async_module;
 
 import 'dart:async';
 
-import 'package:w_flux/w_flux.dart';
+import 'package:meta/meta.dart' show protected;
 import 'package:react/react.dart' as react;
+import 'package:w_flux/w_flux.dart';
 import 'package:w_module/w_module.dart';
 
 class DataLoadAsyncModule extends Module {
+  @override
   final String name = 'DataLoadAsyncModule';
 
   DataLoadAsyncActions _actions;
-  DataLoadAsyncStore _stores;
-
   DataLoadAsyncComponents _components;
-  DataLoadAsyncComponents get components => _components;
+  DataLoadAsyncStore _stores;
 
   DataLoadAsyncModule() {
     _actions = new DataLoadAsyncActions();
     _stores = new DataLoadAsyncStore(_actions);
     _components = new DataLoadAsyncComponents(_actions, _stores);
   }
 
-  Future onLoad() async {
+  @override
+  DataLoadAsyncComponents get components => _components;
+
+  @override
+  @protected
```

Comment:
Are we aware of any additional overhead with using the 'meta' package or is it only at play prior to transpilation?